src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -126,6 +126,17 @@ public ResponseEntity<String> updateUserPassword(@Valid @RequestBody ChangePassw
         return new ResponseEntity<>(getMessageSource(ResponseMessagesHandler.PositiveMessage.CHANGED_USER_PASSWORD), HttpStatus.ACCEPTED);
     }
 
+    @PutMapping(value = ApiKey.USER_SERVICE_RESET_PASSWORD, produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.TEXT_PLAIN_VALUE})
+    @Operation(summary = "Reset a user password by email")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "202", description = "ACCEPTED"),
+            @ApiResponse(responseCode = "400", description = "BAD REQUEST"),
+            @ApiResponse(responseCode = "404", description = "NOT FOUND")})
+    public String resetUserPassword(@Valid @RequestBody ChangeEmailRequest email) {
```

Comment:
In my opinion ChangedEmailReguest is not good idea for using here maybe you should create new request with name ChangePasswordRequest or use just String